### ISSUE TITLE
Show summary of other available Buoys (gathered by Lighthouse) somewhere in Buoy's own interface

### ISSUE THREAD

**Issue by fabacab** [betterangels/buoy#48]
This is a thought - Could we add an a landing page that, at some point after a Buoy alert has been triggered, those in crises are provided with a list of available resources provided by Lighthouses. Those who have triggered an alert could select particular types of resources from a list and receive the location and contact information of that Lighthouse. This would allow users to search with a resource filter. Is this already noted in an issue? Couldn't find one but not sure.

**Comment by fabacab** [comment]
This could also be available to responders.

**Comment by fabacab** [comment]
What do you mean by "a list of available resources provided by Lighthouses"?

**Comment by fabacab** [comment]
I was wrong about the definition of Lighthouse. I was thinking of a lighthouse as a Buoy-enabled website, but, rather, it is a central directory of Buoy-enabled websites. (Yes?)

So with that correction in mind, the idea is this: If there is a central directory of Buoy-enabled websites, maybe administrators of Buoy-enabled websites could also have the option to offer information to that central repository (lighthouse) about the resources they provide as an organization or individual. The repository would store the resources and associated providers in a database. 

Resources could be anything that someone in a crisis situation may need: shelter, a ride or pick-up, on-location backup, a monetary donation, clothing, childcare, immediate advice about legal rights, etc.

A search function could be added to Lighthouses that would allow users to view a list of resources or input their own needed resource and be provided with a filtered a list of Buoy-enabled websites hosted by individuals or organizations providing those resources. 

After finding the resource they need, the end user would create a username and password on that admin's website. They would then add the admin as one of the members of their team.

The end user could then contact the admin (and any other individual on that server who has offered themselves up as a public provider?) about help accessing said resources.

Does this make sense? Basically, it is a way to search for community help based on the resources you need.

**Comment by fabacab** [comment]
Oh, yeah. That is totally on the roadmap. :)

A good example of this that exists today is https://The-Federation.info. This the federated social Web's version of "Lighthouse," in that it is a single site that collects statistics from other sites that run instances of federated social media server software (like Diaspora or HubZilla and so on). This allows individuals like you or me to have one central place to get relevant information about each pod, and to give us the information we need to choose a pod on which to make an account.

Another similar model is the [Tor Atlas](https://atlas.torproject.org/), which provides an interface to the decentralized Tor network of entry, exit, and relay nodes that a given user can choose to use when they make their own connections through the Tor network.

So, basically, yes. What you are proposing was already on the project roadmap (insofar as that roadmap exists in my brain), we just didn't have an issue ticket for it. Now we do. :)

If you want to further define and scope out what Lighthouse should do, the best thing to do would be to draft out the specs in the wiki, perhaps on [this page](https://github.com/meitar/better-angels/wiki/Project-Proposal-Lighthouse).

**Comment by fabacab** [comment]
Sweet, thanks. :)
